resources/views/pages/import.blade.php

```diff
@@ -21,8 +21,10 @@
             <div class="form-check">
                 <label class="form-check-label">
                     <!-- @TODO - make "checked" status a variable that has a default -->
-                    <input checked name="import-type" class="form-check-input" type="checkbox" value="turbovote">
+                    <input name="import-type" class="form-check-input" type="checkbox" value="turbovote">
                     TurboVote Import
```

Comment:
this is totally nitpicky but maybe we can add a `<br>` here so each checkbox is on its own line?

Reply:
@chloealee added!